packages/paypal-js/types/script-options.d.ts

```diff
@@ -21,6 +21,7 @@ interface PayPalScriptQueryParameters {
 }
 
 interface PayPalScriptDataAttributes {
+    env?: "sandbox" | "production";
```

Comment:
@ravishekhar I'm still seeing the name as `env` in the diff. Can we update to `environment`?

Reply:
Ah! I see my git push got denied earlier. Just did that again. Thanks for catching this.